jetstream/config.py

```diff
@@ -329,7 +325,11 @@ def resolve(self, spec: "AnalysisSpec") -> List[Summary]:
 
             pre_treatments = []
             for pt in params.pop("pre_treatments", []):
-                ref = PreTreatmentReference(pt)
+                if isinstance(pt, str):
+                    ref = PreTreatmentReference(pt, {})
```

Comment:
As mentioned below, this only works for lists where all elements are string or all elements are dicts. Mixed type lists are not supported by `toml`